### ISSUE TITLE
Word hangs when opening documents that have been edited using speller version 3.0.

### ISSUE THREAD

**Issue by albbas** [TinoDidriksen/spellers#21]
After Divvun 4.0 was installed, this problem appeared.

Documents  created after 4.0 was installed do not have this problem. Installing Divvun-sme-86.176.10407.msi did not help.

Windows 7
Microsoft Office Home and Business 2013
Version: 15.0.4787.1002
Locale se-FI

**Comment by AinoV** [comment]
[litteraatti 1.docx](https://github.com/TinoDidriksen/spellers/files/113680/litteraatti.1.docx)
This is an example of a file that I'm unable to open.

**Comment by TinoDidriksen** [comment]
Works for me, and all pages get checked.
Did you uninstall the previous speller? The new package does not automatically remove the old one yet, due to a limitation in msitools.

**Comment by AinoV** [comment]
And a screenshot of an error message I received trying to open a Finnish document in Word:
![printscreen](https://cloud.githubusercontent.com/assets/17023025/12746017/3e4aec3e-c9a5-11e5-94d7-de968606ec9d.jpg)
(Free translation: "Word has detected problems with the spell checker se-fi add-in. If the problems persist, disable the add-in and check for updates available. Do you want to disable the add-in now?")

**Comment by AinoV** [comment]
Yes, I've removed the old version, there's only one installed on my computer at the moment.
